tests/pyccel/test_pyccel.py

```diff
@@ -727,6 +761,14 @@ def test_multiple_results(language):
 def test_elemental(language):
     pyccel_test("scripts/decorators_elemental.py", language = language)
 
+#------------------------------------------------------------------------------
+@pytest.mark.gpu
+def test_hello_kernel():
+    types = str
+    pyccel_test("scripts/kernel/hello_kernel.py",
+            dependencies = ("scripts/kernel/func.py") ,
+            language="cuda", output_dtype=types)
```

Comment:
You shouldn't ask Pyccel to compile this code yet as there is no wrapper yet. You need to pass `compile_with_pyccel = False`